homeassistant/components/assist_pipeline/pipeline.py

```diff
@@ -167,6 +167,16 @@ async def _async_create_default_pipeline(
     )
 
 
+async def async_create_default_pipeline(hass: HomeAssistant) -> None:
+    """Create a default pipeline.
+
+    The default pipeline will use the homeassistant conversation agent and the
+    default stt / tts engines.
+    """
+    pipeline_data: PipelineData = hass.data[DOMAIN]
+    await _async_create_default_pipeline(hass, pipeline_data.pipeline_store)
+
+
```

Comment:
I think we should allow passing in something to this function to ensure the pipeline is created with cloud + homeassistant + cloud